New article "Use the Edge Add-ons badge to promote your add-on" (#3550)
MicrosoftDocs/edge-developer: microsoft-edge/extensions/publish/marketing-guidelines.md

Author: link the 1st occurrence of "Microsoft Edge Add-ons" in each h2 section:

at [Microsoft Edge Add-ons]( url )

---
# Marketing guidelines for Microsoft Edge Add-ons
<!-- https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines -->
<!-- focus concretely on "badge", though potentially can broaden later to cover add'l assets; at that time, can use more abstract umbrella wording -->

<!-- lexicon:
Microsoft Edge Add-ons - mainly means the site
Edge Add-ons - short form not currently used

Microsoft Edge Add-ons badge
Edge Add-ons badge - short form not currently used
-->

To promote your Microsoft Edge extension, use the Microsoft Edge Add-ons badge in your websites, documentation, tutorials, and other services or integrations, to ensure consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.

The Microsoft Edge Add-ons badge is provided to you as a set of `.png` files.

<!-- ====================================================================== -->
## The Microsoft Edge Add-ons badge
<!-- https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines#store-badges -->

The Microsoft Edge Add-ons badge is a visual asset designed to help you promote your Microsoft Edge Add-ons, such as extensions, and build user trust.

> ![The Microsoft Edge Add-ons badge](./marketing-guidelines-images/microsoft-edge-add-ons-badge.png)

Author: kept as-is

The Microsoft Edge Add-ons badge lets you showcase your extensions with Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-ons badge is intended exclusively for promoting add-ons (such as extensions, themes, or web apps) that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.

Unlike generic Microsoft [Store badges](/windows/apps/publish/app-marketing-guidelines#store-badges), the Microsoft Edge Add-ons badge provides a distinct, trusted visual cue that connects your add-on directly to the Microsoft Edge ecosystem.

> Use the provided badge `.png` files that are in the `.zip` file (below), rather than creating your own image files.

Author: ```suggestion
```

---
title: Marketing guidelines for Microsoft Edge Add-ons
description: Marketing guidelines for publishing an extension, theme, or web app, such as using badges, at Microsoft Edge Add-ons.
author: MSEdgeTeam
ms.author: msedgedevrel
ms.topic: conceptual
ms.service: microsoft-edge
ms.date: 08/13/2025
---
# Marketing guidelines for Microsoft Edge Add-ons

> <!-- todo: is this content copied (then modified) from a webpage? which? -->

Author: is this content copied (then modified) from a webpage? which?

that could affect eg moving sections around; affects how heavily to alter this article

@G-u-r-j-e-e-t

Author: upstream doc worked yesterday, 404 now

your PR [Comment](https://github.com/MicrosoftDocs/edge-developer/pull/3550/files/df2baa3be93b5de944d77bf2df56e97e2a24d5c9#r2281501767) contains /en-in/ link:

https://learn.microsoft.com/en-in/windows/apps/publish/app-marketing-guidelines - 404
https://learn.microsoft.com/en-us/windows/apps/publish/app-marketing-guidelines - 404
https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines - 404

@G-u-r-j-e-e-t  fyi

<!-- todo: global in this article: lexicon:
the Microsoft Edge Add-on badge
the Microsoft Edge add-on badge
the Microsoft Edge add-ons badge
the Microsoft Edge Add-ons badge
the Featured badge
-->

<!-- todo: global in Extensions docs: which?
add-ons (such as extensions, themes, or web apps)
add-ons (such as extensions, themes, or apps)
-->

> These guidelines outline how to use Microsoft Edge brand assets (such as badges and logos) across websites, documentation, tutorials, and other services or integrations.<!-- todo: ", especially the [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) site"?  is this article specifically for the Microsoft Edge Add-ons site, or other sites as well? -->  They ensure consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.

Author: "and logos" - currently this article has almost no points regarding logos - strike "and logos?"

@G-u-r-j-e-e-t

> These guidelines outline how to use Microsoft Edge brand assets (such as badges and logos) across websites, documentation, tutorials, and other services or integrations.<!-- todo: ", especially the [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) site"?  is this article specifically for the Microsoft Edge Add-ons site, or other sites as well? -->  They ensure consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.

Author: especially for the [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) site?
is this article specifically or 99% for the Microsoft Edge Add-ons site, or often for other sites as well?

@G-u-r-j-e-e-t

Items at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/) include extensions, themes, and web apps.

> See also:

Author: does this article have anything at all to do with the Featured badge?  
https://learn.microsoft.com/en-us/microsoft-edge/extensions/#the-featured-badge

If not, I might omit this link and only have it at end of article

* [The Featured badge](../index.md#the-featured-badge) in _Overview of Microsoft Edge extensions_.

<!-- ====================================================================== -->
## The Microsoft Edge Add-on badge
<!-- todo:
is the "Microsoft Edge Add-on(s) badge" the same thing as "the Featured badge"?
mention [The Featured badge](../index.md#the-featured-badge)? -->

<!-- todo: .png -->

A Microsoft Edge Add-on badge is a visual asset designed to help you promote Edge Add-ons and build user trust.

The Microsoft Edge Add-on badge lets you showcase your extensions with Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-on badge is intended exclusively for promoting add-ons (such as extensions, themes, or web apps) that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.

Unlike generic Microsoft Store badges,<!-- todo: .png(s) --> the Microsoft Edge Add-on badge provides a distinct, trusted visual cue that connects your add-on directly to the Microsoft Edge ecosystem.
<!-- todo: which?

> generic Microsoft Store badges - what's meant? show

Author: url showing a generic Microsoft Store badge(s)?

https://apps.microsoft.com

@G-u-r-j-e-e-t

the Microsoft Store logo - show, if relevant here
-->

Use the provided badges in the `.zip` file (below), rather than creating your own image files.

<!-- ====================================================================== -->
## Minimum size for badges

To ensure that the badge text remains readable, display it at a minimum size of 32px.  Avoid using smaller sizes.

> When scaling the badge, always maintain its aspect ratio; do not stretch or compress the badge image.  This helps preserve visual quality and reinforces your product's Credibility.<!-- todo: credibility? -->

Author: is Credibility a specialized tech term here, eg a metric?  else lowercase

@G-u-r-j-e-e-t

These badges are available in many languages.

<!-- ====================================================================== -->
## Badge usage guidelines
<!-- todo: what kind of badges?
the Microsoft Edge Add-on badge
the Featured badge
-->

Always use the official badge artwork provided by Microsoft.

Do not create your own version, alter the text color, extract the Microsoft Edge logo, or modify the badge in any way.

Do not animate, rotate, tilt, stretch, compress, or apply gradients or shadows to the badge.

Do not use the badge as a decorative element in any context.

The badge must always be an active, clickable link that directs users to your extension page at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).

Do not use the badge in static marketing materials, such as posters, flyers, or non-clickable images in presentations or videos.

The badge is intended solely to facilitate direct access to your content on the Edge Add-ons platform.

<!-- ====================================================================== -->
## License for Microsoft Marks (badges and logos)

> _Microsoft Marks_ refers to the Microsoft Edge badges and logos<!-- todo: strike "and logos?"  this article is not at all about how to use a logo --> (assets) described above.

Author: strike "and logos?" this article is not at all about how to use a logo

@G-u-r-j-e-e-t

ms.author: msedgedevrel
ms.topic: conceptual
ms.service: microsoft-edge
ms.date: 08/13/2025
---
# Marketing guidelines for the Microsoft Edge Add-ons badge
<!-- https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines -->

To promote your Microsoft Edge Add-ons (such as extensions, themes, or web apps), use the Microsoft Edge Add-ons badge in your websites, documentation, tutorials, and other services or integrations.  This ensures consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.

The Microsoft Edge Add-ons badge is a visual asset designed to help you promote your Microsoft Edge Add-ons and build user trust.

![The Microsoft Edge Add-ons badge](./marketing-guidelines-images/microsoft-edge-add-ons-badge.png)

The Microsoft Edge Add-ons badge appears in full color with a dark background, and includes a gray border.  The badge `.png` files are provided in 41 languages.

The Microsoft Edge Add-ons badge lets you showcase your Microsoft Edge Add-ons with Microsoft Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-ons badge is intended exclusively 

> for promoting <!--todo-->add-ons (extensions, themes, or web apps) that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.

Author: ```suggestion
for promoting <!--todo-->add-ons (such as extensions, themes, or web apps) that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.
```
add "such as" here?
@G-u-r-j-e-e-t

Reviewer: for promoting <del>add-ons (extensions, themes, or web apps)</del> Edge Add-ons

<!-- todo: orig draft docx:
for promoting            add-ons and extensions                    that are available for Microsoft Edge. 
-->

You publish an add-on at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/), and then link to your add-on's page at that site from your websites, documentation, tutorials, and other services or integrations.

Unlike generic Microsoft [Store badges](/windows/apps/publish/app-marketing-guidelines#store-badges), the Microsoft Edge Add-ons badge provides a distinct, trusted 
visual cue that connects your add-ons directly to the Microsoft Edge ecosystem.

> <!-- todo: orig draft docx:

Reviewer: visual cue that connects your <del>add-ons</del> Edge Add-ons directly to the Microsoft Edge ecosystem.

<!-- ====================================================================== -->
## Badge usage guidelines

Follow the required specifications for image usage, fonts, and color palette.  Detailed usage examples and guidelines are available in the downloadable PDF:

[![Download button](./marketing-guidelines-images/download-button.png)](https://download.microsoft.com/download/0/7/D/07DF43D4-B1A8-4D38-BC02-4903BB36CEE8/Microsoft_Store_Badge_Guidelines.pdf) **Microsoft Store Badge Guidelines (PDF, English)**

For example, here are some of the guidelines from the PDF:

* Always use the official badge artwork provided by Microsoft.

* Do not create your own version, alter the text color, extract the Microsoft Edge logo, or modify the badge in any way.

* Do not animate, rotate, tilt, stretch, compress, or apply gradients or shadows to the badge.

* Do not use the badge as a decorative element in any context.

* The badge must always be an active, clickable link that directs users to your add-on's page at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).

>   <!-- todo:

Author: check add-ons wording, b/c docx orig said "your extension page"
see todo comments

* Internal preview: https://review.learn.microsoft.com/microsoft-edge/extensions/publish/marketing-guidelines?branch=pr-en-us-3550
* External preview: https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/badges/microsoft-edge/extensions/publish/marketing-guidelines.md

@G-u-r-j-e-e-t

Reviewer: The badge must always be an active, clickable link that directs users to your <del>add-on's</del> product detail page at [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).

  orig draft docx: 
  The badge must always be an active, clickable link that directs users to your extension page on the Add-ons store.
  your item's page
  your Microsoft Edge Add-ons page
  your Add-ons page
  your add-on's page
  your Add-on's page
  your Microsoft Edge Add-on's page
  the page for your add-on
  the page for your Add-on
  the page for your Microsoft Edge Add-on (extension, theme, or web app)
  the page for your Microsoft Edge Add-ons (extensions, themes, or web apps)
  the page for your extension, theme, or web app
  the page for your item
  -->

* Do not use the badge in static marketing materials, such as posters, flyers, or non-clickable images in presentations or videos.

* The badge is intended solely to facilitate direct access to your content on the Microsoft Edge Add-ons platform.

> <!-- todo:

Reviewer: The badge is intended solely to facilitate direct access to your <del>content</del> Edge Add-ons on the <del>Microsoft Edge Add-ons platform</del> [Microsoft Edge Add-ons](https://microsoftedge.microsoft.com/addons/).

---
title: Microsoft Edge Add-ons Marketing Guidelines
description: Microsoft Edge Add-ons Marketing Guidelines.
author: MSEdgeTeam
ms.author: msedgedevrel
ms.topic: conceptual
ms.service: microsoft-edge
ms.date: 08/13/2025
---

> # Microsoft Edge Add-ons Marketing Guidelines

Author: ```suggestion
# Marketing guidelines for Microsoft Edge Add-ons
```
* puts key words first
* lowercase per corp style
   * site name is spelled "Microsoft Edge Add-ons" at the site; this title refers to guidelines for a site, vs. about extensions.

fyi @G-u-r-j-e-e-t

<!-- ====================================================================== -->

> ## Purpose

Author: ```suggestion
```
per this repo, omit Intro heading

These guidelines outline how to use Microsoft Edge brand assets across websites, documentation, tutorials, and other services or integrations.  They ensure consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.

<!-- ====================================================================== -->
## Microsoft Edge Add-ons Badge

Microsoft Edge Add-ons badge is a visual asset designed to help you promote Edge Add-ons and build user trust.

Microsoft Edge Add-ons badge lets you showcase your extensions with Edge-specific branding.  To prevent confusion with the broader Microsoft Store, the Microsoft Edge Add-ons badge is intended exclusively for promoting add-ons and extensions that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser

> Unlike generic Microsoft Store badges — the Microsoft Edge Add-ons badge provides a distinct, trusted visual cue that connects your add-ons such as extensions, webapps directly to the Edge ecosystem

Author: ```suggestion
Unlike generic Microsoft Store badges, the Microsoft Edge Add-ons badge provides a distinct, trusted visual cue that connects your add-ons (such as extensions, themes, or web apps) directly to the Microsoft Edge ecosystem.
```

Author: ```
The Microsoft Edge Add-on badge lets you showcase your extensions with Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-on badge is intended exclusively for promoting add-ons (such as extensions, themes, or web apps) that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.

Unlike generic Microsoft Store badges<!-- todo: .png showing those badges --> — the Microsoft Edge Add-on badge provides a distinct, trusted visual cue that connects your add-on directly to the Microsoft Edge ecosystem.
```

<!-- ====================================================================== -->
#### Minimum sizes

* To ensure the badge text remains readable, display it at a minimum size 32px.  Avoid using smaller sizes.  When scaling the badge, always maintain its aspect ratio—do not stretch or compress it.  This helps preserve visual quality and reinforces your product's Credibility

These badges are available in 41 languages.  Please follow the required specifications for image usage, fonts, and color palette.  Detailed usage examples and guidelines are available in the downloadable PDF.

> [Add working Download button: Microsoft Store Badge Guidelines (PDF, English)]

Author: which website to put .zip file & .pdf file for downloading?  
There was some discussion for a previous PR.

@captainbrosset @leahmsft @G-u-r-j-e-e-t

Author: Where to put the couple download files for users to download?  Similar to the solution we used w/ Utkarsh, you (or I, if I had the two files) could _create a PR in the extensions repo_, to add a dir like /badges/ or /assets/:

https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension-samples

per its Readme https://github.com/microsoft/MicrosoftEdge-Extensions, that repo contains:
* Code
* Issues
* Discussions

@G-u-r-j-e-e-t

Author: closing Outdated comment.
highly likely that as a good solution, we'll upload PDF & ZIP to Extensions repo eg in new /assets/ dir, comparable to /TestCrxPackages/ dir precedent, but w/ better dir structure.

@G-u-r-j-e-e-t @ut-karsh @leahmsft @captainbrosset

<!-- ====================================================================== -->
## All Badge images

* Download the complete set of badge images (PNG formats) from the link below.

[Add working Download button: Microsoft Edge Badge Guidelines (PDF, English)]

<!-- ====================================================================== -->
## Badge usage Tips and Guidelines

* **Always use the official badge artwork provided by Microsoft.**
* **Do not** create your own version, alter the text color, extract the Microsoft Edge logo, or modify the badge in any way.
* **Do not** animate, rotate, tilt, stretch, compress, or apply gradients or shadows to the badge.
* **Do not** use the badge as a decorative element in any context.

> * The badge **must always be an active, clickable link** that directs users to your extension page on the Add-ons store.

Author: The badge must always be an active, clickable link that directs users to your extension 
page at [Microsoft Edge Add-ons]( https://microsoftedge.microsoft.com/addons/ ).

---
title: Marketing guidelines for the Microsoft Edge Add-ons badge
description: Use the Microsoft Edge Add-ons badge in your websites, documentation, or tutorials when publishing an extension, theme, or web app at the Microsoft Edge Add-ons site.  # key words before col 158
author: MSEdgeTeam
ms.author: msedgedevrel
ms.topic: conceptual
ms.service: microsoft-edge
ms.date: 08/13/2025
---

> # Marketing guidelines for the Microsoft Edge Add-ons badge

Author: changing.  this docset is for task oriented more than for policies (or end-user)

<!-- https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines -->

To promote your Microsoft Edge Add-ons (such as extensions, themes, or web apps), use the Microsoft Edge Add-ons badge in your websites, documentation, tutorials, and other services or integrations.  This ensures consistent, accessible, and compliant representation of the Microsoft Edge Add-ons brand.

The Microsoft Edge Add-ons badge is a visual asset designed to help you promote your Microsoft Edge Add-ons and build user trust.

> ![The Microsoft Edge Add-ons badge](./marketing-guidelines-images/microsoft-edge-add-ons-badge.png)

Author: changed from:
the Microsoft Edge Add-ons badge
to:
the Get it from Microsoft Edge badge

The Microsoft Edge Add-ons badge appears in full color with a dark background, and includes a gray border.  The badge `.png` files are provided in 41 languages.

The Microsoft Edge Add-ons badge lets you showcase your extensions with Edge-specific branding.  To prevent confusion with the broader [Microsoft Store](https://apps.microsoft.com), the Microsoft Edge Add-ons badge is intended exclusively for promoting add-ons (such as extensions, themes, or web apps) that are available for Microsoft Edge.  This badge helps users clearly identify content that is built specifically for the Microsoft Edge browser.

Unlike generic Microsoft [Store badges](/windows/apps/publish/app-marketing-guidelines#store-badges), the Microsoft Edge Add-ons badge provides a distinct, trusted visual cue that connects your add-on directly to the Microsoft Edge ecosystem.

The Microsoft Edge Add-ons badge is not the same as the Microsoft brand logo, and they cannot be used interchangeably.

<!-- ====================================================================== -->
## Download the badge image files

Download the complete set of badge `.png` files.

> <!-- todo: upload the .zip file to either:

Author: captainb asked in forum

> <!-- todo: upload the .zip file to either:

Author: Outdated.  [fresh](https://github.com/MicrosoftDocs/edge-developer/pull/3550/files#r2301562335)

<!-- ====================================================================== -->
## License to Microsoft Marks
<!-- https://learn.microsoft.com/windows/apps/publish/app-marketing-guidelines#license-to-microsoft-marks -->

_Microsoft Marks_ means the Microsoft Edge Add-ons badge and Microsoft Edge logos.  To use these assets:
* Your product or content must reference Microsoft Edge or its features.
* You must follow the logo usage specifications described in the [Microsoft Store Badge Guidelines](https://download.microsoft.com/download/0/7/D/07DF43D4-B1A8-4D38-BC02-4903BB36CEE8/Microsoft_Store_Badge_Guidelines.pdf).
* Microsoft reserves the right to revoke usage at any time.

Microsoft is the sole owner of the Microsoft Marks and associated goodwill, and the sole beneficiary of the goodwill associated with your use of the Microsoft Marks.  Microsoft may revoke this license at any time and at its sole discretion.

<!-- ====================================================================== -->
## See also

> * [The Featured badge](../index.md#the-featured-badge) in _Overview of Microsoft Edge extensions_.

Author: removed link from list
